.eslintrc.js

```diff
@@ -5,8 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-const path = require('path');
-const customImportResolver = path.resolve('./eslintImportResolver');
+const customImportResolver = require.resolve('./eslintImportResolver');
```

Comment:
new beta version of intellij tries to be too clever for its own good and failed the resolution here. This fixes it, I snuck it in 😀

Reply:
/cc @segrey (I don't have time to open up a proper bug report on youtrack, sorry!)

Reply:
Actually any path resolution that resolves modules should go through `require.resolve` (to include the extension and work with PnP :D)

Reply:
Sorry for being slow. Thanks, reproduced (https://youtrack.jetbrains.com/issue/WEB-37116).
/cc @undeadcat